modules/maxquant/lfq/main.nf

```diff
@@ -0,0 +1,43 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process MAXQUANT_LFQ {
+    tag "$meta.id"
+    label 'process_long'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::maxquant=2.0.1.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/maxquant:2.0.1.0--py39hdfd78af_2"
+    } else {
+        container "wombatp/maxquant-pipeline:dev"
```

Comment:
Is there no `quay.io` biocontainer for this?

Reply:
There is a quay.io container, but it unfortunately does not work

Reply:
what's the problem with it? Maybe the issue can be solved in the bioconda recipe.

Reply:
Took longer to find the reason: the biocontainers  does not work with symbolic links but I have no clue why my container does as it is also based on the bioconda package. Maybe because the base image (the nfcore one) in my container contains the necessary bash features. I will push our image to quay and check whether this works.

Reply:
I just find it strange that the singularity container works then? It is built from exactly the same source afaik. 

If there are some basic system features missing for the program to work in the container, it is possible to activate the "extended base image" in the bioconda recipe: 
https://bioconda.github.io/contributor/troubleshooting.html?highlight=extended#using-the-extended-image

Reply:
I now tried quite a few options and still have it failing when using the biocontainer. It works with my own container that uses the same bioconda package. It might not be the symbolic links but more an error coming from a non-working build of the container from the bioconda package. 

The singularity container also fails and has failed before. Or am I wrong?

As I still want to build a few more (easier) modules, I suggest to either accept this one (after the necessary fixes) with the flaw missing a biocontainer or I run it locally in the meanwhile.